portfolio/src/main/webapp/script.js

```diff
@@ -29,5 +29,31 @@ function addRandomFact() {
 
   // Add it to the page.
   const factContainer = document.getElementById('fact-container');
-  factContainer.innerText = fact;
+  factContainer.innerText = fact; 
+}
+
+/** 
+ * Picks a caption to display depending on the image clicked
+ */
+function displayCaption(img) {
+  const id = img.id;
+  const imageFacts = 
+  {
+	'chinese-phoenix': `This technically isn't finished yet! A lot of outlining
+        still needs to be added`,
+    'gothic-cat': `This piece is a special type of embroidery called 
+        'Blackwork'.`,
+    'headshot': `This picture was taken in my living room with a
+        phone camera.`,
+    'mit-latin': `This picture was taken at the biggest collegiate competition
+        of the year: MIT Open!`,
+    'scout-sitting': `I love him, but this cat is totally evil.`,
+    'scout-window': `Scout turned four years old this year!`,
+    'selfie': `I'm pretty sure this was originally a Snapchat.`,
+    'tufts-standard': `That's me with my dance partner, Cho, who graduated this
+    year. :( We took this picture because we thought it'd be funny.`
```

Comment:
Can you keep the indentation consistent?

Reply:
Sure, sorry! I've been having some trouble with the auto-indent feature in the cloud shell editor and it messed up the spacing if I wasn't careful. I've finally figured out how to set the indent length to 2, though, so I'm hoping it should be easier to keep consistent now!